docs/src/inputs/radio/radio.jsx

```diff
@@ -1,568 +1,65 @@
+import RadioSamplePill from './examples/radioSamplePill';
+/* eslint-disable import/no-named-default, import/extensions */
+import { default as rootDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/inputs/radio/radio';
+/* eslint-enable import/no-named-default, import/extensions */
+
+export default function ElementsRadio() {
+    const {
+        description,
+        displayName,
+    } = rootDoc;
+
+    return (
+        <Main page={camelCase(displayName)}>
+            <Main.Content>
+
+                <MarkdownContainer>
+                    <Typography
+                        className="description"
+                        variant="body1"
                     >
-                        <Radio.Item
-                            id="block_name--element_name-1"
-                            label="Option 01"
-                            tabIndex={0}
-                        />
-
-                        <Radio.Item
-                            id="block_name--element_name-2"
-                            label="Option 02"
-                            tabIndex={0}
-                        />
+                        {description}
+                    </Typography>
 
-                        <Radio.Item
-                            id="block_name--element_name-3"
-                            label="Option 03"
-                            tabIndex={0}
-                        />
-                    </Radio>
-                    <br />
-                    <br />
-
-                    <Radio
-                        checked={pillDisableValue}
-                        name="pillOnChangeSample"
-                        onChange={this.onPillDisableChange}
-                        pill
-                        multi
-                    >
-                        <Radio.Item label="Option 01" />
-                        <Radio.Item label="Option 02" />
-                        <Radio.Item label="Option 03" />
-                    </Radio>
-                    <br />
-                    <br />
-
-                    <Radio
-                        checked={2}
-                        disable
-                        name="pillDisabledSample"
-                        pill
-                    >
-                        <Radio.Item label="Option 01" />
-                        <Radio.Item label="Option 02" />
-                        <Radio.Item label="Option 03" />
-                    </Radio>
-                    <br />
-                    <br />
-
-                    <Highlighter customStyle={{ marginBottom: '44px', marginTop: '44px' }}>
-                        {radioGroupPillSample}
-                    </Highlighter>
-                </Main.Content>
-            </Main>
-        );
-    }
+                    <Heading anchorLink="children" variant="h2">
+                        Radio
+                    </Heading>
+
+                    <Typography variant="body1">
+                        A simple radio.
```

Comment:
Change "A simple radio" to "General use"

Reply:
done!